fitbenchmarking/parsing/ivp_parser.py

```diff
@@ -0,0 +1,86 @@
+"""
+This file implements a parser for the IVP data format.
+"""
+from collections import OrderedDict
+
+import importlib
+import inspect
+import os
+import sys
+import numpy as np
+
+from fitbenchmarking.parsing.fitbenchmark_parser import FitbenchmarkParser
+from fitbenchmarking.utils.exceptions import ParsingError
+
+import_success = {}
+try:
+    from scipy.integrate import solve_ivp
+    import_success['ivp'] = (True, None)
+except ImportError as ex:
+    import_success['ivp'] = (False, ex)
```

Comment:
We don't need this as a dictionary anymore. In fact this was only here as we couldn't assume the parser would fail based on 1 import exception. With the new implementation, given that the parser is imported only after it is selected we can just let it raise the import error and let the factory handle it (line 61 of the factory).